base/docs/basedocs.jl

```diff
@@ -1144,6 +1144,10 @@ In function declarations, and optionally in calls, `;` separates regular argumen
 While constructing arrays, if the arguments inside the square brackets are separated by `;`
 then their contents are vertically concatenated together.
 
+`;` in first position inside of parentheses can be used to construct a named
+tuple in a flexible way. The same `(; ...)` syntax on the left side of an
```

Comment:
I think this might actually read slightly better without the "in a flexible way". Could give an example instead? Looks good otherwise!

Reply:
I removed "in a flexible way" and added an example of property destructuring. I also added a mention of `;;` and `;;;` concatenation an a corresponding example.